sdk/eventhub/event-hubs/README.md

```diff
@@ -1,259 +1,198 @@
-@azure/event-hubs
-================
+# Azure Event Hubs client library for Javascript
 
-Azure Event Hubs is a scalable event processing service that ingests and processes large volumes of events and data, with low latency and high reliability. More information about Azure Event Hubs can be found over [here](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features).
+Azure Event Hubs is a scalable event processing service that ingests and processes large volumes of events and data, with low latency and high reliability.
 
-This sdk provides a convenient way to interact with the Azure Event Hubs service.
+Use the client library for Azure Event Hubs in your Node.js application to
 
-## Pre-requisite ##
-- **Node.js version: 6.x or higher.** 
-- We would **still encourage you** to install the latest available LTS version at any given time from https://nodejs.org. **It is a good practice to always install the latest available LTS version of node.js.**
-- Installing node.js on **Windows or macOS** is very simple with available installers on the [node.js website](https://nodejs.org). If you are using a **linux based OS**, then you can find easy to follow, one step installation instructions over [here](https://nodejs.org/en/download/package-manager/).
+- Send events to an Event Hub
+- Receive events from an Event Hub
```

Comment:
`Event Hub` -> `Event Hub instance` ?
How do the docs refer to this?

Reply:
Docs refer `event hub`
https://docs.microsoft.com/en-us/azure/event-hubs/

Reply:
In that case lets remove the "an" there.